packages/replay-orchestrator-launcher/src/index.ts

```diff
@@ -58,42 +60,41 @@ export const executeScheduledTestRunChunk = async (
   options: Omit<
     ExecuteScheduledTestRunChunkOptions,
     "logLevel" | "chromeExecutablePath"
-  >
+  >,
 ): Promise<InProgressTestRunChunk> => {
   const logger = log.getLogger(METICULOUS_LOGGER_NAME);
   const bundleLocation = await fetchAsset(
-    EXECUTE_SCHEDULED_TEST_RUN_BUNDLE_PATH
+    EXECUTE_SCHEDULED_TEST_RUN_BUNDLE_PATH,
   );
 
   // eslint-disable-next-line @typescript-eslint/no-require-imports
   return (await require(bundleLocation)).executeTestRunChunk({
     ...options,
-    chromeExecutablePath: getChromiumExecutablePath(),
+    chromeExecutablePath: await getChromiumExecutablePath(),
     logLevel: logger.getLevel(),
   });
 };
 
 export const executeTestRun = async (
-  options: Omit<ExecuteTestRunOptions, "logLevel" | "chromeExecutablePath">
+  options: Omit<ExecuteTestRunOptions, "logLevel" | "chromeExecutablePath">,
 ): Promise<ExecuteTestRunResult> => {
   const logger = log.getLogger(METICULOUS_LOGGER_NAME);
   const bundleLocation = await fetchAsset(
-    "replay/v3/execute-test-run.bundle.js"
+    "replay/v3/execute-test-run.bundle.js",
   );
 
   // eslint-disable-next-line @typescript-eslint/no-require-imports
   return (await require(bundleLocation)).executeTestRun({
     ...options,
-    chromeExecutablePath: getChromiumExecutablePath(),
+    chromeExecutablePath: await getChromiumExecutablePath(),
     logLevel: logger.getLevel(),
   });
 };
 
-const getChromiumExecutablePath = () => {
-  // replay-orchestrator-launcher has a puppeteer dependency, which ensures
-  // that Chromium gets downloaded at the Yarn/NPM pre-install stage. We need to pass
-  // the reference to this specific version of Chromium to the replay bundle, since
-  // the replay bundle may use a slightly different version of Puppeteer, which
-  // may have a slightly different default Chromium path/version.
-  return executablePath();
+const getChromiumExecutablePath = async (): Promise<string> => {
+  // Ensure browser is available using our shared browser installer.
+  // The browser installer will prioritize Chrome versions that are compatible with
+  // the replay bundles to minimize version mismatch warnings. If the preferred
+  // version is unavailable, it will fall back to puppeteer-core's compatible version.
+  return await ensureBrowser();
```

Comment:
This function is identical to `ensureBrowser`, can we drop it and just use that?